docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -1442,21 +1448,35 @@ If the universe has more than three nodes, execute a `destroy --base_dir=<path t
 
 ### Create a single-node universe
 
-Create a single-node universe with a given [base directory](#base-directory). Note the need to provide a fully-qualified directory path for the `base_dir` parameter.
+Create a single-node universe with a given [base directory](#base-directory). You need to provide a fully-qualified directory path for the `base_dir` parameter.
+
+By default, yugabyted uses IPv4. To start a universe with an IPv6 address, specify `::1` for the `--advertise_address`.
```

Comment:
This sentence is not needed. It's not a question of default - you either provide an ipv4 or ipv6 address. "::1"  doesn't indicate an ipv6 address, it is an *example of* an ipv6 address, the ipv6 equivalent of localhost. cc @nchandrappa